spec/builtin/net/ip_range_enum_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/net/ip_range_enum'
+
+describe Ronin::Recon::Net::IPRangeEnum do
+  describe "#process" do
+    context "when there are ip in range" do
+      let(:ip_range) { Ronin::Recon::Values::IPRange.new('192.168.0.1/2') }
+
+      it "must yield IP values" do
+        yielded_values = []
+
+        subject.process(ip_range) do |value|
+          yielded_values << value
+        end
+      end
+    end
+
+    context "when there is no ip in range" do
```

Comment:
I don't think this edge-case should exist? A `Values::IPRange` will always cause IPs to be yielded. Even if the IP range is `A.B.C.D/32` it will only yield `Values::IP.new('A.B.C.D')`.